.pre-commit-config.yaml

```diff
@@ -123,3 +123,13 @@ repos:
       language: python
       verbose: true
       require_serial: true
+
+  - repo: https://github.com/sqlfluff/sqlfluff
+    rev: 2.3.5  # Or latest stable version
+    hooks:
+      - id: sqlfluff-lint
+        name: SQLFluff Linter
+        entry: sqlfluff lint
+        language: python
+        types: [sql]
+        args: [--dialect=postgres]
```

Comment:
Do you need this here and in the .sqlfluff file?
I'd hope it reads from that file so we can keep it all in one place.

Reply:
Feel free to update! Just wanted to get the ball rolling and make this easier to land. Right now it seems like pre-commit may be struggling to work with the change so the PR may require a bit more work.